src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RightTriangle extends Rectangle {
```

Comment:
The 'RightTriangle' class should not extend 'Rectangle' as it restricts the 'RightTriangle' to always be a kind of 'Rectangle', which conceptually may not be true for all cases. Consider using an interface to define common behavior instead.